.bazelrc.common

```diff
@@ -78,7 +80,8 @@ test:debug --test_output=streamed --test_strategy=exclusive --test_timeout=9999
 # The node process will break before user code starts and wait for the debugger to connect.
 run:debug --define=VERBOSE_LOGS=1 -- --node_options=--inspect-brk
 # The following option will change the build output of certain rules such as terser and may not be desirable in all cases
-build:debug --compilation_mode=dbg
+# It will also output both the repo cache and action cache to a folder inside the repo
+build:debug --compilation_mode=dbg --show_result=1 --disk_cache=bazel-cache/disk-cache --repository_cache=bazel-cache/repository-cache
```

Comment:
Similar to your other PR, shouldn't this be in the bazel directory?

Reply:
The idea is to get that one in after all the others be merged so I will just use what we set on the others PRs, but good catch. I've changed it to reflect the others.